locations/spiders/bnp_paribas_fortis_be.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+import scrapy
+from chompjs import chompjs
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours, sanitise_day
+
+
+class BNPParibasFortisBESpider(scrapy.Spider):
```

Comment:
```suggestion
class BnpParibasFortisBESpider(scrapy.Spider):
```
to get around CI?

Reply:
Yep, our class naming system doesn't know about acronyms.